Lite paper faq (#104)
latent-to/developer-docs: docs/learn/synchronized-halving.md

2. **α_in (ALPHA injection)**: ALPHA tokens are minted directly into each subnet's AMM reserve to maintain balanced liquidity on both sides of the pool. This injection is capped by the same global TAO halving schedule as τ_in, ensuring they shrink together proportionally.

3. **α_out (ALPHA outbound)**: ALPHA tokens bound for distribution to network participants (miners, validators, and stakers) accumulate each block throughout an epoch, before being emitted to participants at the epoch's conclusion. Unlike the injections above, injection to **ALPHA out** follow each subnet's own independent ALPHA halving schedule.

**Critical insight**: The first two flows (τ_in and α_in) are synchronized to the global TAO clock, while the third flow (α_out) runs on each subnet's local clock. This timing mismatch is the root cause of the asymmetries described below.

> ### Why α_in must follow TAO halvings

Reviewer: The most critical of all issues is the ALPHA halving compression and that emissions would run out in ~10 years, instead of the intended ~60 years with a 2-year halving schedule.

ALPHA emissions to participants (α_out) remain on each subnet's independent local halving schedule, while injections follow the global TAO schedule.  While synchronizing injections (τ_in and α_in) stabilizes markets, this 'semi-synchronization' creates unintended downstream consequences. This timing split—α_in synchronized globally, α_out running locally—creates a growing imbalance between "ALPHA in AMM pools" versus "ALPHA in participant wallets." This imbalance is the mathematical root cause of all four distortions described below.

## 2) The problem — asymmetries caused by two clocks

The fundamental issue emerges from the timing mismatch between global and local halving schedules. When ALPHA injections (α_in) halve according to TAO's global schedule but ALPHA emissions (α_out) continue on each subnet's independent schedule, the ratio between "ALPHA in AMM pools" versus "ALPHA held by participants" shifts at every global halving event.

This single mathematical drift creates several distinct but related distortions that systematically disadvantage later subnet cohorts:

### 2.1 Liquidity impact on AMMs

> **The mechanism**: As ALPHA emissions (α_out) increasingly dominate relative to ALPHA injections (α_in), the AMM pools receive proportionally less fresh liquidity while participants hold proportionally more tokens. This changes the fundamental dynamics of trading.

Reviewer: ```suggestion
**The mechanism**: As ALPHA emissions (α_out) increasingly dominate relative to ALPHA injections (α_in), the AMM pools receive proportionally less liquidity while participants hold proportionally more tokens. This changes the fundamental dynamics of trading.
```

This makes it progressively harder for newer subnets to maintain healthy liquidity and stable prices, even when participant behavior is identical across cohorts.

### 2.2 Slower root‑share decline

**Background**: The [Root Subnet](../resources/glossary.md#root-subnetsubnet-zero) (Subnet Zero) provides a mechanism for TAO holders to stake in a subnet-agnostic way. Root's influence in any given subnet depends on the total TAO staked there, scaled by the global [TAO‑weight](../resources/glossary.md#tao-weight) parameter (γ, currently 0.18).

**The mechanism**: Root's proportional influence should decline as each subnet's ALPHA supply grows through emissions. However, after TAO halvings, ALPHA injections (α_in) slow down while the TAO-weight parameter (γ) remains constant. This means new ALPHA supply grows more slowly, allowing Root to maintain disproportionate influence for longer periods.

**Cohort impact**: 
- **Early subnets**: Root's influence declined predictably as ALPHA supply grew
- **Later subnets**: Root maintains higher influence for extended periods due to slower ALPHA supply growth

> **Potential solution**: Scale the TAO-weight parameter with each TAO halving (e.g., 0.18 → 0.09 → 0.045 → …). This would keep Root's effective power declining in sync with the overall issuance scale, preserving consistent dilution dynamics across all epochs.

Reviewer: ```suggestion
**Solution**: Scale the TAO-weight parameter with each TAO halving (e.g., 0.18 → 0.09 → 0.045 → …). This would keep Root's effective power declining in sync with the overall issuance scale, preserving consistent dilution dynamics across all epochs.
```

Reviewer: It's definitely what needs to be done if synchronizing

### 2.3 Disadvantageous liquidation

**Subnet deregistration context**: Subnets can be deregistered when they no longer meet economic or operational criteria. When this occurs, the AMM pool is dissolved, and all ALPHA holders redeem their tokens against the pool's remaining TAO reserves.

**Two critical prices**:
- **Spot price**: The current market rate offered by the AMM (TAO reserves ÷ ALPHA reserves)

> - **Liquidation price**: The redemption value if the subnet shuts down (pool's TAO ÷ total outstanding ALPHA held by users)

Reviewer: ```suggestion
- **Liquidation price**: The redemption value if the subnet get deregistered (TAO reserves ÷ total outstanding ALPHA held by users)
```

**The liquidation [haircut](./halvings-problem#haircut-h)**: When ADR > 1, the liquidation price falls below the spot price. This creates a built-in "haircut" risk where ALPHA holders recover less TAO than their tokens' current market value suggests.

**What is a haircut?** A haircut is the discount between what you expect to receive (based on current market price) and what you actually receive during liquidation. For example, if your ALPHA tokens are worth 100 TAO at market price but you only receive 75 TAO during subnet liquidation, you've suffered a 25% haircut.

**Cohort impact examples**:
- **Early subnets** (ADR ≈ 1): Liquidation price matches spot price — no haircut
- **Later subnets** (ADR >> 1): Liquidation price significantly below spot — substantial haircut risk

This asymmetry makes staking in later subnet cohorts fundamentally riskier, even when the underlying subnet performance is identical.

> ### 2.4 Halving "speed‑up" (interval compression)

Reviewer: This should be 2.1 by order, it's the most relevant issue

## 3) Toward a Solution: Exploring a synchronized design

### 3.1 Core change — one clock for both

The proposed solution directly addresses the root cause by eliminating the timing mismatch:

**Synchronization principle**: Set every subnet's ALPHA emissions (α_out) to halve on the same global blocks as TAO halvings, rather than following independent local schedules.

**Unified timing**: Now all three flows—TAO injections (τ_in), ALPHA injections (α_in), and ALPHA emissions (α_out)—shrink together proportionally at each global halving event.

### 3.2 Immediate benefits

Synchronization eliminates all four distortions at their mathematical source:

1. **Cohort-invariant liquidity**: Selling a fixed fraction of rewards produces identical AMM slippage impact across all subnet cohorts, regardless of when they were created.

2. **Predictable root dynamics**: Root Subnet influence follows a stable dilution trajectory when the TAO-weight parameter (γ) is scaled alongside issuance (e.g., 0.18 → 0.09 → 0.045).

3. **Elimination of liquidation haircuts**: The liquidation price matches the spot price for all cohorts (ADR = 1), removing built-in disadvantages for later subnets.

> 4. **Stable halving intervals**: ALPHA halving intervals maintain their intended 2-year spacing across all epochs—no compression or acceleration.

Reviewer: They wouldn't maintain their intended 2-year schedule, ALPHA halvings would essentially be eliminated and ALPHA emissions would instead halve with TAO every 4 years

### 3.3 Key implications and trade-offs

While synchronization solves the core asymmetry problems, it introduces some changes to the current system:

#### Supply implications

> **Reduced maximum supply for some cohorts**: The current system's interval compression effectively accelerates some subnets beyond their intended pace. Synchronization removes this artificial acceleration, resulting in lower total ALPHA supply for cohorts that would have benefited from compression.

Reviewer: Not only reduced maximum supply but also (and this already happens actually), higher maximum supplies than 21 million ALPHA.

**Predictable supply curves**: Each subnet's maximum ALPHA supply becomes calculable based on its registration timing relative to the global TAO supply, creating transparent and predictable tokenomics.

#### Long-term considerations

**Emission endpoint**: Once halvings reduce per-block rewards below the smallest unit (RAO = 10^9 TAO), minting would naturally stop unless token precision is increased.

**Two tail options**:

> 1. **Hard-stop tail**: Emissions end at the final discrete halving when rewards reach the RAO floor. Simple and finite, but creates a discontinuous endpoint.

Reviewer: That's not good because it would have to increase the TAO supply above 21 million

2. **Zeno-halving tail**: Increase token precision (e.g., from 9 to 18 decimal places) so halvings can continue smoothly below the current minimum unit. Rewards asymptotically approach zero without ever stopping, maintaining the same finite total supply with smoother mathematical properties.

#### Parameter adjustments

**TAO-weight scaling**: The TAO-weight parameter (γ) should be halved at each TAO halving (0.18 → 0.09 → 0.045 → …) to keep Root Subnet influence properly aligned with the shrinking issuance scale across all epochs.

## 5) Further considerations

### Timing urgency

The asymmetries described above begin manifesting immediately after the first TAO halving and compound with each subsequent halving. Early implementation preserves fairness across subnet cohorts, while delayed implementation creates permanent advantages for early subnets and disadvantages for later ones.

### Transition approach

Several implementation strategies could be considered:
- **Immediate synchronization**: All existing subnets switch to the global halving schedule

> - **Grandfathering**: Existing subnets continue on local schedules, new subnets use global schedule

Reviewer: > - **Grandfathering**: Existing subnets continue on local schedules, new subnets use global schedule

That wouldn't really be a good solution, sure it could be decided by the subnets, but they would reach their supply in ~10 years instead of the intended ~60 years and then they would also benefit from the liquidity and liquidation etc.
